src/components/TaskItem.js

```diff
@@ -1,11 +1,18 @@
 import React from "react";
+import {connect} from "react-redux";
 
-function TaskItem(props) {
+function TaskItems(props) {
```

Comment:
Small issue here. TaskItem does not need to be turned into a container. 
This component should have just been left as it was